BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -257,19 +275,25 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
   }
   
   @objc private func tappedRewardsSummaryButton() {
-    let contentView = walletView.contentView
     let rewardsSummaryView = walletView.rewardsSummaryView
     
     let isExpanding = rewardsSummaryView.transform.ty == 0
+    transformRewardSummaryView(isExpanding: isExpanding, animated: true)
+  }
+  
+  private func transformRewardSummaryView(isExpanding: Bool, animated: Bool) {
```

Comment:
intent/apple naming: `transformRewardSummaryView` → `setRewardsSummaryVisible(_ visible: Bool, animated: Bool = false)`

Reply:
Updated. Kept that name as in either case it is still visible, only a change in height.